redstone/client.py

```diff
@@ -460,6 +468,21 @@ def unwrap(self, key_id, ciphertext, aad=None):
         resp = self._action(key_id, "unwrap", data)
         return base64.b64decode(resp['plaintext'].encode())
 
+    def create_import_token(self, expiration, max_retrievals):
+        data = {'expiration': expiration,
+                'maxAllowedRetrievals': max_retrievals}
+
+        resp = self.session.post("%s/api/v2/import_token" % self.endpoint_url,
+                                 json=data)
+
+        self._validate_resp(resp)
+        return
```

Comment:
You should return the response here, but I don't have API references to work from so...